### PR TITLE
Era5 uk biobank

### PR DESCRIPTION
**Pull request by gsy19971111**

### CODE REVIEW

File: jointContribution/AI_Climate_disease/config/era5_ukb.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (HydrogenSulfate) [inline jointContribution/AI_Climate_disease/config/era5_ukb.yaml]: 多余的换行请删除

File: jointContribution/AI_Climate_disease/main.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+def export(cfg: DictConfig):
+    pass
+
+
+def inference(cfg: DictConfig):
+    pass
+
+
+@hydra.main(version_base=None, config_path="./config", config_name="era5_ukb.yaml")# joint contribution文件夹下
```
Comment (HydrogenSulfate) [inline jointContribution/AI_Climate_disease/main.py]: 不必要的注释请删除

File: jointContribution/AI_Climate_disease/main.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    elif cfg.mode == "infer":
+        inference(cfg)
+    else:
+        raise ValueError(
+            f"cfg.mode should in ['train', 'eval', 'export', 'infer'], but got '{cfg.mode}'"
+        )
+
+
+if __name__ == "__main__":
+    main()
```
Comment (HydrogenSulfate) [inline jointContribution/AI_Climate_disease/main.py]: 文件末尾补充一个换行

File: jointContribution/AI_Climate_disease/model.py [hunk 4]
```diff
@@ -0,0 +1,523 @@
+# -*- coding: utf-8 -*-
+import math
+from typing import Optional, Tuple
+import numpy as np
+import paddle
+import paddle.nn as nn
+import paddle.nn.functional as F
+from paddle.io import Dataset, DataLoader
+# 选中第 0 张 GPU；如有多卡改成 'gpu:1' 等
+# paddle.set_device('gpu:0')
```
Comment (HydrogenSulfate) [inline jointContribution/AI_Climate_disease/model.py]: 删除

File: jointContribution/AI_Climate_disease/结项报告.md
Comment (HydrogenSulfate) [inline jointContribution/AI_Climate_disease/结项报告.md:1]: 这个文件可以不用放到PR里

File: ppsci/loss/bce.py
Comment (HydrogenSulfate) [inline ppsci/loss/bce.py:1]: 在loss.md中添加新增的loss类

File: ppsci/data/dataset/era5_land_dataset.py
Comment (HydrogenSulfate) [inline ppsci/data/dataset/era5_land_dataset.py:1]: 这个文件建议放到案例文件夹AI_Climate_disease/下吧，代码开头注册到ppsci.dataset即可

``` py
from ppsci.data import register_to_dataset
class YourDataset(paddle.io.Dataset):
    pass

register_to_dataset(YourDataset)
```

**Comment by HydrogenSulfate** [comment]
@gsy19971111 今天合入的PR比较多，还麻烦顺便解决一下冲突
